persistence.js

```diff
@@ -65,6 +65,7 @@ MongoPersistence.prototype._setup = function () {
     } else {
       var urlParsed = urlModule.parse(that._opts.url)
       var databaseName = that._opts.database || (urlParsed.pathname ? urlParsed.pathname.substr(1) : undefined)
+      databaseName = databaseName.substr(databaseName.lastIndexOf('/') + 1)
```

Comment:
How does the attached test verify this? `that._opts_database` is set, so there would be no `/`.